frontend/src/config/test.schema.json

```diff
@@ -0,0 +1,120 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "test.schema.json",
+  "title": "Test / Lab Order",
+  "description": "Information recorded for a test / lab order",
+  "type": "object",
+  "properties": {
+    "testingLabSpecimenID": {
+      "description": "(is this a unique test ID?)",
+      "type": "string"
+    },
+    "orderTestDate": {
+      "description": "",
+      "type": "string",
+      "format": "date-time"
+    },
+    "testResultCoded": {
+      "description": "Test result - need to change to ENUM SNOMED-CT Qualitative Values:",
+      "type": "string"
+    },
+    "specimenCollectionDateTime": {
+      "description": "Test result date",
+      "type": "string",
+      "format": "date-time"
+    },
+    "orderingProviderID": {
+      "description": "Ordering provider name; NPI",
+      "type": "string"
+    },
+    "patientInfo": {
+      "description": "time of test information for this patient",
+      "type": "object",
+      "properties": {
+        "patientID": {
+          "description": "Unique patient identifier",
+          "type": "string"
+        },
+        "firstTest": {
+          "description": "is this their first test I assume?",
+          "type": "boolean"
+        },
+        "symptomatic": {
+          "description": "symptomatic per CDC",
+          "type": "string",
+          "enum": ["YES", "NO", "UNK - Unknown"]
+        }
+      }
+    },
+    "facilityInfo": {
+      "description": "facility information at the time of test",
+      "type": "object",
+      "properties": {
+        "testingLabName": {
+          "description": "Name of Testing Location",
+          "type": "string"
+        },
+        "testingLabID": {
+          "description": "CLIA Number",
+          "type": "string"
+        },
+        "orderingProvider": {
+          "description": "Ordering provider",
+          "type": "object",
+          "properties": {
+            "lastName": {
+              "description": "",
+              "type": "string"
+            },
+            "firstName": {
+              "description": "",
+              "type": "string"
+            },
+            "street": {
+              "description": "",
+              "type": "string"
+            },
+            "zipCode": {
+              "description": "",
+              "type": "string"
+            },
+            "phone": {
+              "description": "",
+              "type": "string",
+              "pattern": "^(\\([0-9]{3}\\))?[0-9]{3}-[0-9]{4}$"
+            }
+          }
+        },
+        "orderedTestCode": {
+            "description": "LOINC CODES ENUM TO BE ADDED",
+            "type": "string"
+        },
+        "specimenSourceSiteCode" {
```

Comment:
missing a colon

Reply:
woops how did I not notice this in vscode, fixing